src/set_theory/game/pgame.lean

```diff
@@ -570,6 +570,8 @@ end
 `relabelling x y` says that `x` and `y` are really the same game, just dressed up differently.
 Specifically, there is a bijection between the moves for Left in `x` and in `y`, and similarly
 for Right, and under these bijections we inductively have `relabelling`s for the consequent games.
+
+In ZFC, relabellings would indeed be the same games.
```

Comment:
Is there a reason we don't quotient by `relabelling`? If we could but don't, then this isn't a type theory vs ZFC thing, but a lean vs literature thing.